massa-storage/Cargo.toml

```diff
@@ -6,7 +6,10 @@ edition = "2021"
 
 [dependencies]
 parking_lot = "0.12"
+massa_logging = { path = "../massa-logging" }
 massa_models = { path = "../massa-models" }
+serde_json = "1.0"
+tracing = "0.1"
```

Comment:
Do we need tracing and json if we use massa-logging ?

Reply:
Yes, see https://github.com/massalabs/massa/issues/2543

Reply:
mhh ok.